core nodes: Use-case and examples for YAML (#990)
flowforge/website: src/_includes/core-nodes/yaml-use-case.md

> 
> YAML, or Yet-Another-Markup-Language, is often used as a data markup style for
> with is human readable. In Node-RED, the YAML node is used to convert data between
> YAML format and JavaScript objects (JSON). The YAML node is a core component of Node-RED
> that comes pre-installed.

Author: ```suggestion
YAML, or Yet-Another-Markup-Language, is often used as a data markup style as it
is human readable. In Node-RED, the YAML node is used to convert data between
YAML format and JavaScript objects (JSON).
```
